.gitignore

```diff
@@ -5,3 +5,5 @@
 /docs/_build
 /media/
 notes.md
+__pycache__/
+*.pyc
```

Comment:
Perhaps leveraging one of GitHub's .gitignore templates would be useful here.

Reply:
I have personal templates I just didn't copy over yet due to, well, this all being very alpha :) 

(my systems all have global ignores configured)